src/Uno.UI/UI/Xaml/UIElement.Android.cs

```diff
@@ -170,7 +170,7 @@ internal void ResetFramePostArrange()
 
 			ViewCompat.SetClipBounds(this, physicalRect);
 
-			SetClipChildren(NeedsClipToSlot);
+			SetClipToPadding(NeedsClipToSlot);
```

Comment:
As this is somehow "risky", should we add a feature flag for that?

Reply:
It will in my next PR.